stone/backends/js_client.py

```diff
@@ -104,5 +107,17 @@ def _docf(self, tag, val):  # pylint: disable=unused-argument
         Callback used as the handler argument to process_docs(). This converts
         Stone doc references to JSDoc-friendly annotations.
         """
-        # TODO(kelkabany): We're currently just dropping all doc ref tags.
+        # TODO(kelkabany): We're currently just dropping all doc ref tags ...
+        # NOTE(praneshp): ... except for versioned routes
+        if tag == 'route':
+            if ':' in val:
+                val, version = val.split(':', 1)
+                version = int(version)
+            else:
+                version = 1
+            url = fmt_url(self.cur_namespace.name, val, version)
```

Comment:
By my understanding the string returned is to reference a method not the URL. Should we use `fmt_func` instead?

Reply:
Hmm. I am going to leave this one as a TODO. The reason is that the previous (empty) implementation of this function led to url type representations.

Reply:
It's OK. I'm not sure but I think for SDK users, URLs should be transparent to them. They don't need to know the underlying details.